### PR TITLE
Add 0. Getting Started

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

Add "0. Getting Started"

 

### Details

- Inspired by https://github.com/dask/dask-tutorial/blob/main/00_overview.ipynb

### CODE REVIEW

**Comment by akaszynski** [comment]
Starting to look at this. Thanks for getting this started!

**Comment by tkoyama010** [comment]
This is a draft, but I would be grateful for a review.

**Comment by tkoyama010** [comment]
- [ ] Check the link of methods and functions.

**Comment by tkoyama010** [comment]
- [ ] Add the title and the abstract of the section.

File: 0_getting_started.ipynb [hunk 1]
```diff
@@ -0,0 +1,195 @@
+{
```
Comment (tkoyama010) [inline 0_getting_started.ipynb:1]: <p>Add links to each section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyvista/pyvista-tutorial/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

**Pull request merged**